server/controllers/finance/reports/debtors/annual-clients-report.js

```diff
@@ -130,7 +103,9 @@ function getDebtorGroupMovements(fiscalYearId, currencyId, rate, hideLockedClien
       LEFT JOIN period_total pt ON dg.account_id = pt.account_id
       LEFT JOIN account ac ON ac.id = pt.account_id
       LEFT JOIN period p ON p.id = pt.period_id
-    WHERE pt.fiscal_year_id = ? ${hideLockedClients ? hiddenClientsCondition : ''}
+    WHERE pt.fiscal_year_id = ?
+      ${hideLockedClients ? hiddenClientsCondition : ''}
+      ${includeCashClients ? '' : excludeCashClientsCondition}
     GROUP BY pt.account_id;
```

Comment:
Is it possible to have ORDER BY ? because in the report data are not ordered, we can order by ac.number or dg.groupName

Reply:
`ORDER BY ac.number, dg.name ASC` can do the job